RecoTracker/FinalTrackSelectors/interface/TrackMVAClassifier.h

```diff
@@ -85,7 +88,7 @@ class TrackMVAClassifier : public TrackMVAClassifierBase {
     void computeMVA(reco::TrackCollection const & tracks,
 		    reco::BeamSpot const & beamSpot,
 		    reco::VertexCollection const & vertices,
-		    MVACollection & mvas) const final {
+		    MVACollection & mvas) final {
```

Comment:
why const being removed?
(here and elsewhere)

Reply:
See comment in
https://github.com/cms-sw/cmssw/blob/06dd36bf33cebea3247ad17d8afd8d8bf0cf7131/RecoTracker/FinalTrackSelectors/interface/TrackMVAClassifier.h#L37-L39
It's either remove `const`, or we construct `std::map<string, double>` for each track. Or we could preserve the constness of interface by marking the member 
https://github.com/cms-sw/cmssw/blob/06dd36bf33cebea3247ad17d8afd8d8bf0cf7131/RecoTracker/FinalTrackSelectors/plugins/TrackLwtnnClassifier.cc#L75
as `mutable` (which should work in practice since the EDProducer is stream, even it it is a bit ugly).

Reply:
perhaps this is a better choice, making an exception for a special case, while still keeping the interface possibly usable as const in all other places.

Reply:
another solution to keep things const is to implement ```TrackLwtnnClassifier::computeMVA ``` with lwt::ValueMap cached per event .. or even add a template parameter here to create a cache object and pass it to the MVA::operator call